### PR TITLE
Add transform logSuggestionForFailureResponse

### PR DESCRIPTION
**Pull request by JiayuW2**
## Problem
Following up comments on this PR: https://github.com/aws/language-servers/pull/476
There are probably other places where we log some suggestion after a failure message.

## Solution
Addressed comments on the above mentioned PR. Rephrase the suggestion message and put it into a function.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jonlouie — APPROVED** [review]

File: server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts [hunk 1]
```diff
@@ -439,4 +435,20 @@ export class TransformHandler {
...
     }
+
+    logSuggestionForFailureResponse(request: GetTransformRequest, job: TransformationJob, failureStates: string[]) {
+        let status = job?.status ?? PollTransformationStatus.NOT_FOUND
+        let reason = job?.reason ?? ''
+        if (failureStates.includes(status)) {
+            let suggestion = ''
+            if (reason.toLowerCase().includes('build validation failed')) {
+                suggestion =
+                    'Please close Visual Studio, delete the directories where build artifacts are generated (e.g. bin and obj), and try running the transformation again.'
```
Comment (ege0zcan) [inline server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts:446]: Do we expect this server to be only used in visual studio and no other IDE? Otherwise I would reword this to `Please close your IDE`

Comment (JiayuW2) [reply server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts:446]: Yes, it's only used in visual studio.

**Review by ege0zcan — APPROVED** [review]
